### PR TITLE
docs(bodiless-table): Update Table Component documentation

### LINKED ISSUE #1048: [DOC] Table Component Documentation
## User Story

AS AN end user of the Bodiless documentation,
I WANT to understand how to implement the Table Component both from code and from the GUI,
SO THAT I can share the information with the business and development team.

## Acceptance Criteria

1. The Documentation is updated to include how a user can implement and use the Table Component.

### PR DESCRIPTION
**Pull request by misterjones**
## Changes

* Update Table Component documentation.
  * Add "Content Editor Details."
  * Update "Site Builder Details."
  * Add a number of image assets.

---

Resolves #1048

### CODE REVIEW

**Review by e2thex — APPROVED** [review]

File: packages/bodiless-table/README.md [hunk 1]
```diff
@@ -1,16 +1,219 @@
...
+
+- **Table:** A table with a header row.
+- **Table Header First Column:** A table with a header row _and_ a header column.
+
+### Add a Table
+
+To add a Table Component to your page:
+
+01. While in [Edit Mode](/ContentEditorUserGuide/#edit-mode), select a [Flow
+    Container](/Components/FlowContainer), and, within its context menu, under "Component," click
```
Comment (Guilherme-Almeida-Zeni) [inline packages/bodiless-table/README.md:23]: Hello @misterjones ,

Should the comma be inside the double quotes? I mean, `"Component,"` or `"Component",` ?

File: packages/bodiless-table/README.md [hunk 2]
```diff
@@ -1,16 +1,219 @@
...
+    **Add** (or **Swap**, if you're replacing a component).
+01. In the "Insert Component" modal, under "Type," check the "Table" checkbox, and select the table
```
Comment (Guilherme-Almeida-Zeni) [inline packages/bodiless-table/README.md:25]: Hello @misterjones ,

Same here, should it be `"Insert ... Type,"` or `"Insert ... Type",` ?

File: packages/bodiless-table/README.md [hunk 3]
```diff
@@ -1,16 +1,219 @@
...
+#### Add a Row
+
+To add a body (i.e., non-header/non-footer) row to your table:
+
+01. Select the row above where you would like to add a new row.
+    - **Note:** Selecting a _header_ or _footer_ row will not provide the correct context menu.
+      - If you want to add a body row directly below a header row, see the note below.
+      - If you want to add a body row directly below a footer row, you can't; only more footer
+        rows can be added below a footer row.
+01. From the context menu, under "Row," click **Add**.
```
Comment (Guilherme-Almeida-Zeni) [inline packages/bodiless-table/README.md:90]: Hello @misterjones ,

Same here, should be `"Row,"` or `"Row",` ?

File: packages/bodiless-table/README.md [hunk 4]
```diff
@@ -1,16 +1,219 @@
...
+Both of the default Table Component variants already have a header row, but you're able to add as
+many as you require.
+
+Header rows can only be positioned in the header (upper) region of a table — they cannot be _added_
+or _moved_ below it. Body rows and footer rows cannot be positioned above a header row.
+
+To add a header row to your table:
+
+01. Select the header row above where you would like to add a new header row.
+01. From the context menu, under "Header Row," click **Add**.
```
Comment (Guilherme-Almeida-Zeni) [inline packages/bodiless-table/README.md:114]: Hello @misterjones ,

Same here, should be `"Header Row,"` or `"Header Row",` ?

Comment (Guilherme-Almeida-Zeni) [inline packages/bodiless-table/README.md:114]: Hello @misterjones ,

The number `01` appears twice in lines 113/114, is this correct? (if it was meant to be a numbered list, maybe the sequence 01/02 might make more sense to separate topics). Also, this seems to be happening in several places.

File: packages/bodiless-table/README.md [hunk 5]
```diff
@@ -1,16 +1,219 @@
...
+variants include a footer row by default, but you may add as many as you require.
+
+Footer rows can only be positioned in the footer (lower) region of a table — they cannot be _added_
+or _moved_ above it. Body rows and header rows cannot be positioned below a footer row.
+
+To add a footer row to your table:
+
+- If your table doesn't have a footer row yet:
+  01. You may select anywhere on the table.
+  01. From the context menu, under "Table," click **Footer** (designated with an "add" icon
```
Comment (Guilherme-Almeida-Zeni) [inline packages/bodiless-table/README.md:139]: Hello @misterjones ,

Should be `"Table,"` or `"Table",` ?

File: packages/bodiless-table/README.md [hunk 6]
```diff
@@ -1,16 +1,219 @@
...
+
+- Columns are moved to the right.
+  - I.e., a moved column will swap places with the column to its immediate right.
+- Rows are moved down.
+  - I.e., a moved row will swap places with the row directly below it.
+
+To move a column or row:
+
+01. Select a table cell in the column or row that you would like to reorder.
+01. From the context menu, under "Column" or "Row," click **Move**.
```
Comment (Guilherme-Almeida-Zeni) [inline packages/bodiless-table/README.md:194]: Hello @misterjones ,

Should be `"Row,"` or `"Row",` ?

File: packages/bodiless-table/README.md [hunk 7]
```diff
@@ -1,16 +1,219 @@
...
+styling of your site, this may remain the case. If you need to discern whether a row is a body row
+or a footer row, simply select it, and, in its context menu, you will see either a "Row" section or
+a "Footer Row" section, revealing the type of row that it is.
+
+### Delete Columns and Rows
+
+To delete a column or row from your table:
+
+01. Select a table cell in the column or row that you would like to remove.
+01. From the context menu, under "Column" or "Row," click **Delete**.
```
Comment (Guilherme-Almeida-Zeni) [inline packages/bodiless-table/README.md:170]: Hello @misterjones ,

Should be `"Row,"` or `"Row",` ?

File: packages/bodiless-table/README.md [hunk 8]
```diff
@@ -1,16 +1,219 @@
...
+
+- If your table doesn't have a footer row yet:
+  01. You may select anywhere on the table.
+  01. From the context menu, under "Table," click **Footer** (designated with an "add" icon
+      (**+**)).  
+      ![Table section in context menu](./assets/TableSectionContextMenu.jpg ':size=80')  
+      A footer row will be added to the bottom of your table.
+- If your table already has one or more footer rows:
+  01. Select the footer row above where you would like to add a new footer row.
+  01. From the context menu, under "Footer Row," click **Add**.
```
Comment (Guilherme-Almeida-Zeni) [inline packages/bodiless-table/README.md:145]: Hello @misterjones ,

Should be `"Footer Row,"` or `"Footer Row",` ?

**Review by Guilherme-Almeida-Zeni — APPROVED** [review]

**Pull request merged**